test/circuits_sim/device/aht20_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule CircuitsSim.Device.AHT20Test do
+  use ExUnit.Case
+
+  alias CircuitsSim.I2C.I2CServer
+
+  test "simple usage" do
+    device = CircuitsSim.Device.AHT20.new()
+    init_arg = [bus_name: "i2c-1", address: 0x38, device: device]
+    I2CServer.start_link(init_arg)
+
+    I2CServer.send_message("i2c-1", 0x38, {:set_humidity_rh, 12.3})
+    I2CServer.send_message("i2c-1", 0x38, {:set_temperature_c, 32.1})
```

Comment:
Could you make an alias at the top for `CircuitsSim.Device.AHT20` and update all of the other I2CServer calls here to call AHT20 like below?

```suggestion
    AHT20.set_humidity_rh("i2c-1", 0x38, 12.3)
    AHT20.set_humidity_rh("i2c-1", 0x38, 32.1)
```